rush-plugins/rush-init-project-plugin/src/terminal.ts

```diff
@@ -7,17 +7,29 @@ import {
 
 export class TerminalSingleton {
   private static _instance: Terminal | undefined;
+  private static _verboseEnabled: boolean = false;
   private constructor() {}
-  public static getInstance(verboseEnabled?: boolean): Terminal {
+  public static getInstance(): Terminal {
     if (!TerminalSingleton._instance) {
       TerminalSingleton._instance = new Terminal(
         new ConsoleTerminalProvider({
-          verboseEnabled,
+          verboseEnabled: TerminalSingleton._verboseEnabled,
         })
       );
     }
     return TerminalSingleton._instance;
   }
+  public static setVerboseEnabled(enabled?: boolean): void {
```

Comment:
Is it better to only receive a boolean type, like `setVerboseEnable(enabled: boolean)`